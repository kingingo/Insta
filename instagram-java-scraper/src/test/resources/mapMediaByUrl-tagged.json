{
  "graphql": {
    "shortcode_media": {
      "__typename": "GraphImage",
      "id": "1656812236692197349",
      "shortcode": "Bb-LfN7jw_l",
      "dimensions": {
        "height": 809,
        "width": 1080
      },
      "gating_info": null,
      "media_preview": null,
      "display_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-15/e35/23969706_259123051284268_441547961155977216_n.jpg",
      "display_resources": [
        {
          "src": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-15/s640x640/sh0.08/e35/23969706_259123051284268_441547961155977216_n.jpg",
          "config_width": 640,
          "config_height": 479
        },
        {
          "src": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-15/s750x750/sh0.08/e35/23969706_259123051284268_441547961155977216_n.jpg",
          "config_width": 750,
          "config_height": 561
        },
        {
          "src": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-15/e35/23969706_259123051284268_441547961155977216_n.jpg",
          "config_width": 1080,
          "config_height": 809
        }
      ],
      "is_video": false,
      "should_log_client_event": false,
      "tracking_token": "eyJ2ZXJzaW9uIjo1LCJwYXlsb2FkIjp7ImlzX2FuYWx5dGljc190cmFja2VkIjpmYWxzZSwidXVpZCI6ImU2YjQ1MWFhMzZkMDRiYWY4NDZmZTQ1MGQ5ZTVhNDgxMTY1NjgxMjIzNjY5MjE5NzM0OSJ9LCJzaWduYXR1cmUiOiIifQ==",
      "edge_media_to_tagged_user": {
        "edges": [
          {
            "node": {
              "user": {
                "username": "topichaite"
              },
              "x": 0.356,
              "y": 0.9825463535228678
            }
          }
        ]
      },
      "edge_media_to_caption": {
        "edges": [
          {
            "node": {
              "text": "paskendom \u017emoni\u0173 j\u016broj\ud83d\udda4 #concert #lilasirinnomine #kaunas #zalgirioarena #friend #happy #goals #crowded #tbt #fridaynight #memories #amazingtime #vscocam #vsco #lithuanian #lithuania #girls #tagsforlikes #l4l #followme"
            }
          }
        ]
      },
      "caption_is_edited": false,
      "edge_media_to_comment": {
        "count": 0,
        "page_info": {
          "has_next_page": false,
          "end_cursor": null
        },
        "edges": []
      },
      "comments_disabled": false,
      "taken_at_timestamp": 1511727431,
      "edge_media_preview_like": {
        "count": 17,
        "edges": [
          {
            "node": {
              "id": "4983217451",
              "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/23596575_1934128066852675_3144491341476528128_n.jpg",
              "username": "makodeknou"
            }
          },
          {
            "node": {
              "id": "1499124016",
              "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/23421922_127381471264571_5015737832757526528_n.jpg",
              "username": "ozgie99"
            }
          },
          {
            "node": {
              "id": "533234860",
              "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/20590063_461405927548598_4182581238952886272_a.jpg",
              "username": "ryanoosterling"
            }
          },
          {
            "node": {
              "id": "2101901562",
              "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/21985111_773035432880473_2983549115780038656_n.jpg",
              "username": "piwny_swiat_tomka"
            }
          },
          {
            "node": {
              "id": "1466596484",
              "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/22857519_304826989998627_8958409826417770496_n.jpg",
              "username": "margerinemann_666"
            }
          },
          {
            "node": {
              "id": "5065915888",
              "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/22427206_120573628638602_8402668279818092544_n.jpg",
              "username": "klaudyynee"
            }
          },
          {
            "node": {
              "id": "4477743094",
              "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/23279260_1990229934578188_5454066767464759296_n.jpg",
              "username": "kolobsa"
            }
          },
          {
            "node": {
              "id": "174600278",
              "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/22070843_711095609076999_1576600524910755840_n.jpg",
              "username": "mikinnamcgerr"
            }
          },
          {
            "node": {
              "id": "2994217072",
              "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/22580672_162742030979274_4474304999769243648_n.jpg",
              "username": "thatsme_club"
            }
          },
          {
            "node": {
              "id": "2215041254",
              "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/19984519_136555123609131_1084923672570888192_a.jpg",
              "username": "lnsles"
            }
          }
        ]
      },
      "edge_media_to_sponsor_user": {
        "edges": []
      },
      "location": {
        "id": "251482381",
        "has_public_page": true,
        "name": "\u017dalgirio arena",
        "slug": "zalgirio-arena"
      },
      "viewer_has_liked": false,
      "viewer_has_saved": false,
      "viewer_has_saved_to_collection": false,
      "owner": {
        "id": "1371292938",
        "profile_pic_url": "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/22708673_1524703324275010_171053880086364160_n.jpg",
        "username": "aistejakk",
        "blocked_by_viewer": false,
        "followed_by_viewer": false,
        "full_name": "Aist\u0117 Jaku\u010dionyt\u0117",
        "has_blocked_viewer": false,
        "is_private": false,
        "is_unpublished": false,
        "is_verified": false,
        "requested_by_viewer": false
      },
      "is_ad": false,
      "edge_web_media_to_related_media": {
        "edges": []
      }
    }
  }
}