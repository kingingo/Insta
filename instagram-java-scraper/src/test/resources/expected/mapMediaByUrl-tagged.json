{
  "height" : 809,
  "width" : 1080,
  "displayUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-15/e35/23969706_259123051284268_441547961155977216_n.jpg",
  "displayResources" : [ {
    "src" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-15/s640x640/sh0.08/e35/23969706_259123051284268_441547961155977216_n.jpg",
    "width" : 640,
    "height" : 479
  }, {
    "src" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-15/s750x750/sh0.08/e35/23969706_259123051284268_441547961155977216_n.jpg",
    "width" : 750,
    "height" : 561
  }, {
    "src" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-15/e35/23969706_259123051284268_441547961155977216_n.jpg",
    "width" : 1080,
    "height" : 809
  } ],
  "isVideo" : false,
  "shouldLogClientEvent" : false,
  "trackingToken" : "eyJ2ZXJzaW9uIjo1LCJwYXlsb2FkIjp7ImlzX2FuYWx5dGljc190cmFja2VkIjpmYWxzZSwidXVpZCI6ImU2YjQ1MWFhMzZkMDRiYWY4NDZmZTQ1MGQ5ZTVhNDgxMTY1NjgxMjIzNjY5MjE5NzM0OSJ9LCJzaWduYXR1cmUiOiIifQ==",
  "mediaType" : "GraphImage",
  "id" : 1656812236692197349,
  "shortcode" : "Bb-LfN7jw_l",
  "caption" : "paskendom ≈æmoni≈≥ j≈´rojüñ§ #concert #lilasirinnomine #kaunas #zalgirioarena #friend #happy #goals #crowded #tbt #fridaynight #memories #amazingtime #vscocam #vsco #lithuanian #lithuania #girls #tagsforlikes #l4l #followme",
  "commentCount" : 0,
  "commentPreview" : {
    "count" : 0,
    "pageInfo" : {
      "hasNextPage" : false
    }
  },
  "commentsDisabled" : false,
  "captionIsEdited" : false,
  "takenAtTimestamp" : 1511727431000,
  "likeCount" : 17,
  "firstLikes" : [ {
    "id" : 4983217451,
    "username" : "makodeknou",
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/23596575_1934128066852675_3144491341476528128_n.jpg"
  }, {
    "id" : 1499124016,
    "username" : "ozgie99",
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/23421922_127381471264571_5015737832757526528_n.jpg"
  }, {
    "id" : 533234860,
    "username" : "ryanoosterling",
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/20590063_461405927548598_4182581238952886272_a.jpg"
  }, {
    "id" : 2101901562,
    "username" : "piwny_swiat_tomka",
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/21985111_773035432880473_2983549115780038656_n.jpg"
  }, {
    "id" : 1466596484,
    "username" : "margerinemann_666",
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/22857519_304826989998627_8958409826417770496_n.jpg"
  }, {
    "id" : 5065915888,
    "username" : "klaudyynee",
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/22427206_120573628638602_8402668279818092544_n.jpg"
  }, {
    "id" : 4477743094,
    "username" : "kolobsa",
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/23279260_1990229934578188_5454066767464759296_n.jpg"
  }, {
    "id" : 174600278,
    "username" : "mikinnamcgerr",
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/22070843_711095609076999_1576600524910755840_n.jpg"
  }, {
    "id" : 2994217072,
    "username" : "thatsme_club",
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/22580672_162742030979274_4474304999769243648_n.jpg"
  }, {
    "id" : 2215041254,
    "username" : "lnsles",
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/19984519_136555123609131_1084923672570888192_a.jpg"
  } ],
  "location" : {
    "id" : 251482381,
    "name" : "≈Ωalgirio arena",
    "slug" : "zalgirio-arena"
  },
  "owner" : {
    "id" : 1371292938,
    "username" : "aistejakk",
    "fullName" : "Aistƒó Jakuƒçionytƒó",
    "isPrivate" : false,
    "isVerified" : false,
    "blockedByViewer" : "false",
    "followedByViewer" : false,
    "hasBlockedViewer" : false,
    "profilePicUrl" : "https://instagram.fhel3-1.fna.fbcdn.net/t51.2885-19/s150x150/22708673_1524703324275010_171053880086364160_n.jpg",
    "requestedByViewer" : false,
    "isUnpublished" : false
  },
  "viewerHasLiked" : false,
  "viewerHasSaved" : false,
  "viewerHasSavedToCollection" : false,
  "isAdvertising" : false,
  "taggedUser" : [ {
    "username" : "topichaite",
    "x" : 0.356,
    "y" : 0.9825463
  } ],
  "created" : 1511727431000
}